cmd/rook/rook/rook.go

```diff
@@ -177,6 +180,24 @@ func GetOperatorImage(clientset kubernetes.Interface, containerName string) stri
 	return containerImage
 }
 
+func GetOperatorImagePullSecrets(clientset kubernetes.Interface) []v1.LocalObjectReference {
+
+	// If provided as a flag then use that value
+	if len(imagePullSecrets) > 0 {
+		var result []v1.LocalObjectReference
+		for _, name := range imagePullSecrets {
+			result = append(result, v1.LocalObjectReference{Name: name})
+		}
+		return result
+	}
+	// Getting the info of the operator pod
+
```

Comment:
I think the comment above goes with the line below, and this space should go where the comment above is. ???

Reply:
Yes! I will fix this.